sklearn/covariance/_graph_lasso.py

```diff
@@ -1066,6 +1091,7 @@ def fit(self, X, y=None):
                 cv=cv,
                 n_jobs=self.n_jobs,
                 verbose=inner_verbose,
+                params=params,
```

Comment:
I think that we are missing a common test for checking if `grid_scores` uses properly the `params`.
I would advocate to add a small test to check it.

Reply:
We can improve the `test_graphical_lasso_cv_scores` test in this regard.

Reply:
@glemaitre I added a test. Kindly verify if it looks okay.